source/gsoc.md

```diff
@@ -26,10 +26,12 @@ Please watch this page for updates concerning our participating in 2021.
 * Watch [GSoC video presentation](https://www.youtube.com/watch?v=S6IP_6HG2QE)
  for students.
 * Examine the list of [available ASF GSoC 2021 projects](https://s.apache.org/gsoc2021ideas)
-	* Note that you can search the issue tracker for GSoC projects by
+	* <span class="text-primary">**NOTE**</span> that you can search the issue tracker for GSoC projects by
 filtering using the "gsoc2021" label, this allows you to narrow down the list
```

Comment:
BTW, do you think it will be kind to provide any link for `committers/GSoCLinkId.txt` on line 145? Sorry... actually, I do not know where the file is.

Reply:
I guess this is in SVN, not sure why is it here